docs/TUTORIAL.md

```diff
@@ -268,115 +284,95 @@ and how the `reporting_day` variable that you passed is used in the `control.rep
 
 
-9. Run the singular tests:
+8.  Inspect the code in the `use_cases/examples/home_loan_delinquency/dbt/tests`
+    folder, which contains `singular` tests. Notice that the tests in this folder
+    implement a table comparison between the actual results that are output by the
+    `dbt run` command, and the expected results that are saved in the `homeloan_expectedresults`
+    dataset.
+
+
+9.  Run the singular tests:
+
     ```
-    dbt test -s test_type:singular
+    dbt test --profiles-dir profiles/ -s test_type:singular
     ```
 
 
 10. Generate the documentation for the project:
+
     ```
-    dbt docs generate --profiles-dir profiles && dbt docs serve --profiles-dir profiles 
+    dbt docs generate --profiles-dir profiles && \
+    dbt docs serve --profiles-dir profiles
     ```
+    
+11. In the output that you see, search for, and then click, the following URL
+    text: `http://127.0.0.1:8080`
+
+    Your browser opens a new tab that shows the dbt documentation web interface.
+
 
+12. Inspect the lineage of the models and their documentation. You see that the 
+    documentation includes all of the code and the documentation for the models
+    (as specified in the `models/schema.yml` files).
 
-11. In the output that you see, search for, and then click, the following URL text: http://127.0.0.1:8080
     
-    Your browser opens a new tab that shows the dbt documentation web interface.
+13. In Cloud Shell, enter the following:
 
+        Ctrl + c
 
-12. Explore the lineage of the models, and their detailed documentation. You see that  the documentation includes all
-the models’ documentation as specified in the  models/schema.yml files, and all the code of the models.
+      Cloud Shell stops hosting the dbt web interface.
 
-----
-## (Optional) Containerize the transformations
-1. Create a container for the BigQuery data load step, and push the container to Google Container Repository:
-    ```
-    cd ../../../../     # the gcloud command should be executed from the root 
-    gcloud builds submit --config use_cases/examples/home_loan_delinquency/data_load/cloudbuild.yaml
-    ```
-    The Dockerfile in the data_load directory enables this containerization, which simplifies orchestration of the 
-   workflow.
+## Optional: Containerize the transformations
 
+1.   In Cloud Shell, create a container for the Big Query
+     data load step, and push the container to Google Container Repository:
 
-2. Containerize the DBT code for the data transformation step, and push the container to Google Container Repository. 
-    ```
-    gcloud builds submit --config use_cases/examples/home_loan_delinquency/dbt/cloudbuild.yaml
-    ```
-    Containerization helps you to create a package that can be easily versioned and deployed. 
+     ```
+     cd ../data_load/     # Note the Dockerfile in this folder
+     gcloud builds submit --tag $GCR_DATALOAD_IMAGE  # Pushes the image to GCR
```

Comment:
This is now gcloud builds submit --tag $HOMELOAN_GCR_DATALOAD

Reply:
this should be `gcloud builds submit --config use_cases/examples/home_loan_delinquency/data_load/cloudbuild.yaml`